docs/blog/2018-10-16-unstructured-data/index.md

```diff
@@ -0,0 +1,229 @@
+                    <img src={pokemon.sprites.front_default} alt={pokemon.name} />
+                    <p>{pokemon.name}</p>
+                </Link>
+            </li>
+        ))}
+    {...}
+);
+```
+
+#### 5. Create a page [for each Pokémon](https://github.com/jlengstorf/gatsby-with-unstructured-data/blob/0a91d87b9d4d24a0e6b04b33cc271e054b7467b6/gatsby-node.js#L32).
+
+```javascript{11-18}:title=gatsby-node.js
+exports.createPages = async ({ actions: { createPage } }) => {
+  const allPokemon = await getPokemonData(["pikachu", "charizard", "squirtle"])
+
+  // Create a page that lists all Pokémon.
+  createPage({
+    path: `/`,
+    component: require.resolve("./src/templates/all-pokemon.js"),
+    context: { allPokemon },
+  })
+
+  // Create a page for each Pokémon.
+  allPokemon.forEach(pokemon => {
+    createPage({
+      path: `/pokemon/${pokemon.name}/`,
+      component: require.resolve("./src/templates/pokemon.js"),
+      context: { pokemon },
+    })
+  })
+}
+```
+
+#### 6. Create a page [for each ability of each Pokémon](https://github.com/jlengstorf/gatsby-with-unstructured-data/blob/0a91d87b9d4d24a0e6b04b33cc271e054b7467b6/gatsby-node.js#L40).
+
+```javascript{19-26}:title=gatsby-node.js
+exports.createPages = async ({ actions: { createPage } }) => {
+  const allPokemon = await getPokemonData(["pikachu", "charizard", "squirtle"])
+
+  // Create a page that lists all Pokémon.
+  createPage({
+    path: `/`,
+    component: require.resolve("./src/templates/all-pokemon.js"),
+    context: { allPokemon },
+  })
+
+  // Create a page for each Pokémon.
+  allPokemon.forEach(pokemon => {
+    createPage({
+      path: `/pokemon/${pokemon.name}/`,
+      component: require.resolve("./src/templates/pokemon.js"),
+      context: { pokemon },
+    })
+
+    // Create a page for each ability of the current Pokémon.
+    pokemon.abilities.forEach(ability => {
+      createPage({
+        path: `/pokemon/${pokemon.name}/ability/${ability.name}/`,
+        component: require.resolve("./src/templates/ability.js"),
+        context: { pokemon, ability },
+      })
+    })
+  })
+}
+```
+
+For each type of page, we are invoking the `createPage` action, and supplying it with our desired path, React component, and data (as props on the context).
+
+> View the full source code of this example at Jason Lengstorf's ["gatsby-with-unstructured-data" repo](https://github.com/jlengstorf/gatsby-with-unstructured-data). Also check out the ["using-gatsby-data-layer" branch](https://github.com/jlengstorf/gatsby-with-unstructured-data/tree/using-gatsby-data-layer) of that repo, to compare a refactor that uses Gatsby's data layer in the same example.
+
+## When might using "unstructured data" make sense?
+
+You may find this approach useful when using Gatsby's data layer feels a bit too heavy-handed for your project scope.
+
+## The pros of using unstructured data
+
+- The approach is familiar and comfortable, especially if you’re new to GraphQL
```

Comment:
I'd make the first bullet point "when prototyping" and "when new to Gatsby"

That's the main times GraphQL causes friction because people just want to try out Gatsby and get stuck understanding GraphQL + data plugins. We'd want to push people early to start with just fetching data directly and creating pages and only later migrate to data plugins + GraphQL